src/cards/arrow.ts

```diff
@@ -98,6 +99,8 @@ export function arrowEffect(multiShotCount: number, collideFnKey: string, doesPi
             collideFnKey
           }, underworld, prediction);
 
+          GameStatistics.trackArrowFired({ sourceUnit: state.casterUnit }, underworld, prediction);
```

Comment:
I want to rethink how this is being calculated.  The CPU is doing a ton of extra work here.  For every arrow it's creating a new array (via slice) containing 3 existing Stats objects and then incrementing all of them, and then that's done over and over again for each arrow.  Now this may not be noticeable perf-wise, but this is the kind of thing that could become a problem later if it's changed in some way given that it's invoked so much for something that could just be a single if statement somewhere.

Let's just add `arrowsFired` to effectState.aggregator, increment that inside arrowEffect and then check it at the end of the spell.